### PR TITLE
remove indices / attachments during case rebuild

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py [hunk 1]
```diff
@@ -466,6 +466,28 @@ def test_archive_against_deleted_case(self):
...
+        ])
+
+        case_accessors = CaseAccessors(REBUILD_TEST_DOMAIN)
+        case = case_accessors.get_case(child_case_id)
+        self.assertEqual(1, len(case.indices))
+
+        xform.archive()
+
+        case = case_accessors.get_case(child_case_id)
+        self.assertEqual(0, len(case.indices))
```
Comment (benrudolph) [inline corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py]: 👍

**Review by benrudolph — APPROVED** [review]

**Comment by benrudolph** [comment]
no tests were run oddly...

**Comment by snopoke** [comment]
@mkangia When we rebuild a case (for example after archiving a form) we need to clear the indices and attachments in case the forms associated with the case no longer create the index / attachment,

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**